deploy-satellite-update-playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@sthirugn thats fine, but lets make it clear in the README as well, and add instructions on how to use